delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/StorageFileRejection.scala

```diff
@@ -65,6 +69,46 @@ object StorageFileRejection {
         extends FetchAttributeRejection(rejection.loggedDetails)
   }
 
+  /**
+    * Rejection returned when a storage cannot fetch a file's attributes
+    */
+  sealed abstract class CopyFileRejection(loggedDetails: String) extends StorageFileRejection(loggedDetails)
```

Comment:
Could we have it only on the `FileRejection` side ?

Reply:
Ahh as in ditch the existing pattern since it just adds more error types than are necessary? I guess this came from monix types in the past